docs/compass/03-packages-api.md

```diff
@@ -0,0 +1,81 @@
+# Packages API
+
+## Introduction
+
+On Kyma Runtime, every Application is represented as a single Service Class, and every Package of a given Application is represented as a single Service Plan.
+
+![API Packages Diagram](./assets/packages-api.svg)
+
+This document describes API for Packages, which groups multiple API Definitions, Event Definitions and Documents.
```

Comment:
I'd add a little explanation on what API Definitions, Event Definitions and Documents are. Maybe a terminology section?